ui-v2/app/router.js

```diff
@@ -10,7 +11,7 @@ export const routes = {
   // Our parent datacenter resource sets the namespace
   // for the entire application
   dc: {
-    _options: { path: ':dc' },
+    _options: { path: '/:dc' },
```

Comment:
Is this change breaking? should there be a redirect from non-slash to slash?

Reply:
I think both of these are the same, so the functionality remains the same.

It's actually not got a lot to do with the PR, but I have here so I made it all consistent with the rest of the app, I'd guess this slashless version is probably copied from the old v1 UI.